src/pages/home/owners/InvitationsTable.tsx

```diff
@@ -0,0 +1,199 @@
+                </TableCell>
+                <TableCell>
+                  <Stack
+                    sx={ {
+                      flexDirection: "row",
+                      alignItems: "center",
+                      columnGap: 1.5,
+                      whiteSpace: "nowrap",
+                    } }
+                  >
+                    { pictureUrl ? (
+                      <img
+                        style={ {
+                          borderRadius: "50%",
+                          width: "40px",
+                          height: "40px",
+                        } }
+                        src={ pictureUrl }
+                        alt="Profile"
+                      />
+                    ) : (
+                      <Stack sx={ { height: "40px", width: "40px" } }></Stack>
+                    ) }
+                    { firstName && lastName ? `${firstName} ${lastName}` : null }
+                  </Stack>
+                </TableCell>
+                <TableCell>{ role }</TableCell>
+                <TableCell sx={ { textAlign: "end" } }>
+                  { duration
+                    ? convertMillisecondsToSongFormat(duration)
+                    : "--:--" }
+                </TableCell>
+                <TableCell>{ `${royaltyRate}%` }</TableCell>
+                <TableCell>
+                  { status === "Editing" ? (
+                    <Stack
+                      flexDirection="row"
+                      columnGap={ 2 }
+                      justifyContent="end"
+                    >
+                      <IconButton
+                        aria-label={ `Decline ${title} song collaboration` }
+                        onClick={ () => handleDecline(collaborationId) }
+                        sx={ {
+                          backgroundColor: theme.colors.red,
+                          borderRadius: "8px",
+                          "&:hover": {
+                            backgroundColor: theme.colors.red,
+                            opacity: 0.9,
+                          },
+                        } }
+                      >
+                        <CloseIcon sx={ { color: theme.colors.white } } />
+                      </IconButton>
+                      <IconButton
+                        aria-label={ `Accept ${title} song collaboration` }
+                        onClick={ () => handleAccept(collaborationId) }
+                        sx={ {
+                          backgroundColor: theme.colors.green,
+                          borderRadius: "8px",
+                          "&:hover": {
+                            backgroundColor: theme.colors.green,
+                            opacity: 0.9,
+                          },
+                        } }
+                      >
+                        <CheckIcon sx={ { color: theme.colors.white } } />
+                      </IconButton>
+                    </Stack>
+                  ) : (
+                    "ho"
```

Comment:
Is this an acronym? Because I hope it's an acronym.

Reply:
def an acronym haha added a TODO to come back and fix it before it gets merged